testapi.pm

```diff
@@ -181,7 +181,6 @@ sub record_soft_failure {
     bmwqemu::log_call(reason => $reason);
 
     $autotest::current_test->record_soft_failure_result($reason);
-    $autotest::current_test->{dents}++;
```

Comment:
There is another occurrence of this line in `force_soft_failure` which needs to be removed as well.

Reply:
I fixed it.